ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main__tempo-product-ocp-4.14-lp-interop.yaml

```diff
@@ -36,18 +36,18 @@ tests:
-          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"},
-          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"},
-          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": " distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "Tempo e2e tests", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": " distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "Tempo e2e tests", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"}
+          {"step": "build-logs", "failure_type": "pod_failure", "classification": "Tempo operator images", "jira_project": "LPTOCPCI"},
+          {"step": "build-logs", "failure_type": "test_failure", "classification": "Tempo operator images", "jira_project": "TRACING"},
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure provisioning", "jira_project": "LPTOCPCI"},
+          {"step": "install", "failure_type": "pod_failure", "classification": "Tempo operator install", "jira_project": "LPTOCPCI"},
+          {"step": "install", "failure_type": "test_failure", "classification": "Tempo operator install", "jira_project": "TRACING"},
+          {"step": "distributed-tracing-install-*", "failure_type": "pod_failure", "classification": "Distributed tracing operators", "jira_project": "LPTOCPCI"},
+          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING"},
+          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPTOCPCI"},
+          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": " distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "Tempo e2e tests", "jira_project": "LPTOCPCI"},
+          {"step": " distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "Tempo e2e tests", "jira_project": "TRACING"}
```

Comment:
extra spaces in begginning `" distributed-tracing-tests-*"`

Reply:
Fixed